phone_list.py

```diff
@@ -1,14 +1,14 @@
 import sys
 
-arg = sys.argv[1]
+insert = sys.argv[1]
 
-if arg == '1':
+if insert == '1':
```

Comment:
nu ma refeream la numele variabilei, acesta trebuie sa fie ca inainte ci la insasi argumentul cela, adica 1 in cazul nostru , inloc de 1 trebuie sa fie insert